eva-server/pom.xml

```diff
@@ -30,6 +30,7 @@
         <dependency>
             <groupId>org.springframework.boot</groupId>
             <artifactId>spring-boot-starter-data-mongodb</artifactId>
+            <version>1.4.0.RELEASE</version>
```

Comment:
I've made changes to the versions here: https://github.com/EBIvariation/eva-ws/pull/21/commits/ed094d2f0f1c1e38d6d69d9baeb59aebefc11ea3

* I've increased spring-boot-starter-parent version to 1.4.2
* I've manually added the mongo-java-driver dependency with version 2.14.2 to override the spring driver dependency